Orange/widgets/visualize/ownomogram.py

```diff
@@ -1251,7 +1254,11 @@ def get_points_from_coeffs(current_value, coefficients, possible_values):
                sorted_coefficients[i] * sorted_values[i] * (1 - k)
 
     def reset_settings(self):
-        self._reset_settings()
+        with warnings.catch_warnings():
+            # setting target_class_index will trigger this innocent warning
+            warnings.filterwarnings(
+                "ignore", "combo box 'target_class_index' is empty")
+            self._reset_settings()
```

Comment:
@markotoplak, the warning about invalid index is ignored here.

Nomogram is one of the few widgets that implements `reset_settings`. It sets the settings to default values. If there is no data, the default target class, 0, is invalid, but it's OK and harmless. I don't see a way around it.

If I haven't missed anything, this is the only place where this warning is ignored instead of being fixed.
```suggestion
            self._reset_settings()
```
```suggestion
            self._reset_settings()
```